creator/lib/screens/login_screen.dart

```diff
@@ -104,51 +104,53 @@ class _LoginScreenState extends State<LoginScreen> {
     );
   }
 
-  void _navigateBasedOnRole() {
+  void _navigateBasedOnRole() async {
```

Comment:
why use async here?

Reply:
I'll check other issue because this task cannot build app.
https://github.com/KingdomSeekersJP/salon_app/issues/21